cmd/fyne/internal/mobile/build.go

```diff
@@ -159,7 +159,7 @@ func runBuildImpl(cmd *command) (*packages.Package, error) {
 		}
 		if buildRelease {
 			if len(allArchs["ios"]) > 2 {
-				targetArchs = []string{"arm", "arm64"}
+				targetArchs = []string{"arm64"}
```

Comment:
Sadly this (and the change below) removes iOS 32bit, which we currently support.
Both iOS and iOSSimulator targets build arm64 and x86_64, would it not be better if the simulator built arm64 and x86_64, but the ios targeted arm and arm64?

Reply:
I propose to drop the iOS 32bit support in the next release, as we have already done #2487 already.

Reply:
> I propose to drop the iOS 32bit support in the next release, as we have done #2487 already.

I'm not sure about your assertion here - `develop` still builds for iOS 32bit when compiled with Go 1.14.
We should not drop support for a platform as a side effect of another feature.
Additionally whilst we support Go 1.14 we should support not drop support for some of the platforms it builds for.

Reply:
> still builds for iOS 32bit when compiled with Go 1.14.

I don't really understand why iOS 32bit is still targeted. When Apple (as the system manufacturer) is not supporting these devices (i.e. iPhone 5/5c or earlier, iPad 3 or earlier). People may still have these devices (I do, I still have an iPhone 5), but are people still frequently using them today? Building software that targeting for such a device is like fantasy -- meaning the developer must have: a compatible OS X, a compatible Xcode toolchain, and an appropriate physical device -- otherwise, testing can also be an issue.

Claiming the support of these systems may sound quite cool and responsible. But the support can strap the development process and future moving, such as no testing environment, too many version checks, etc.

> We should not drop support for a platform as a side effect of another feature.

I won't framing the argument of "dropping the support" as a side effect of this feature. I'd propose it become a general release goal.

> Additionally whilst we support Go 1.14 we should support not drop support for some of the platforms it builds for.

Honestly, if possible I'd also propose to bump to Go 1.16 for several reasons:

1. 1.14 is not supported by Go team
2. Installing a new go version is not a problem if one follow the Go's official distribution, such as `go install golang.org/dl/go1.16@latest`; `go1.16 download`, then `go1.16` is already an usable `go` command.

Reply:
One more thing: "still builds" is far away from "quite usable". Given the device's actual computation capability, it is reasonable to say that the device may not run a rendering extensive application smoothly and swiftly.

Reply:
> I don't really understand why iOS 32bit is still targeted.

It relates to my previous concerns that Go team are removing support for things too aggressively.

> Building software that targeting for such a device is like fantasy

That seems a little harsh - it is working for some of our developers.

> I won't framing the argument of "dropping the support" as a side effect of this feature. I'd propose it become a general release goal.

Ok, so open a new ticket to hold the discussion, this thread is probably not the right place.

> Honestly, if possible I'd also propose to bump to Go 1.16 for several reasons:

Sorry this is not possible. Our process is to support Go versions that are delivered as standard in up to date operating systems. Debian, for example, packages 1.15 in the current release - their next major release is not for 18 months or more, so we won't update beyond 1.15 until then.

I am sorry that this may seem frustrating - but if the first thing that developers looking to work with Fyne see is some cryptic compile error because we require new APIs then we are not doing a great job for onboarding.

> it is reasonable to say that the device may not run a rendering extensive application smoothly and swiftly.

In my testing the apps work reasonably for many usages. I would worry that if we require the full-on modern CPUs to render well, then there is not much processor left for apps to do their own work ;)